cloudfront/iiif.wellcomecollection.org/terraform/main.tf

```diff
@@ -11,21 +14,17 @@ module "iiif-stage" {
   environment         = "stage"
   acm_certificate_arn = module.cert.arn
 
-  dlcs_lambda_associations = [
-    {
-      event_type = "origin-request"
-      lambda_arn = local.dlcs_path_rewrite_arn_latest
-    }
-  ]
+  origins = local.origins["stage"]
 
-  # Temporary variable to differentiate prod/stage cache behaviour
-  # Defaults to "loris" where unspecified
-  miro_sourced_images_target = "dlcs_space_8"
+  behaviours = local.stage_behaviours
 }
 
 module "iiif-test" {
   source = "./cloudfront_distro"
 
   environment         = "test"
   acm_certificate_arn = module.cert.arn
+
+  origins    = local.origins["test"]
+  behaviours = local.behaviours
```

Comment:
Minor: should this be a local called `test_behaviours`, and defined where the other two variables are? That way all three are defined in the same place, and it's clearer there if you want to change the behaviour of the test distribution.